core/src/main/resources/addonlangs.json

```diff
@@ -23,16 +23,22 @@
     "baseUri":"http://www.wikidata.org",
     "apiUri":"https://www.wikidata.org/w/api.php"
   },
-  "core": {
-    "wikiCode": "core",
-    "name": "Core Directory",
-    "isoCode": "en",
-    "iso639_3": "eng"
-  },
   "commons": {
     "wikiCode": "commons",
     "name": "Commons",
     "isoCode": "en",
+    "iso639_3": "eng",
+    "dbpediaDomain":"commons.dbpedia.org",
```

Comment:
We mainly needed to include the `baseUri` and `apiUri` as they are different from the usual wikipedia URL (and should be commons.wikimedia.org). Otherwise, they would point to commons.wikipedia.org and so would the triples.